src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,212 @@
+                Car car = getCar(resultSet);
+                cars.add(car);
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new RuntimeException("Couldn't find all cars by driver id: " + driverId, e);
+        }
+    }
+
+    private void insertDrivers(Car car) {
+        String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUES(?, ?);";
+        Long driverId = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                driverId = driver.getId();
+                statement.setLong(2, driverId);
+                statement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Couldn't create a relation "
+                    + "between driver with id = " + driverId
+                    + " and car in data base. Car: " + car, e);
+        }
+    }
+
+    private void deleteAllDrivers(Long id) {
+        String query = "DELETE FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't delete driver from"
+                    + " cars data base with id " + id, e);
+        }
+    }
+
+    private Car getCar(ResultSet resultSet) throws SQLException {
+        long id = resultSet.getObject("car_id", Long.class);
+        String model = resultSet.getString("model");
+        Manufacturer manufacturer = getManufacturer(resultSet);
+        return new Car(id, model, manufacturer);
+    }
+
+    private Manufacturer getManufacturer(ResultSet resultSet) throws SQLException {
+        return new Manufacturer(resultSet.getLong("manufacturer_id"),
+                resultSet.getString("name"), resultSet.getString("country"));
+    }
+
+    private List<Driver> getDriversByCar(Long id) {
+        List<Driver> drivers = new ArrayList<>();
+        String query = "SELECT d.* FROM cars_drivers cd "
+                + "JOIN drivers d ON cd.driver_id = d.id "
+                + "WHERE car_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                Driver driver = new Driver(
+                        resultSet.getLong("id"),
+                        resultSet.getString("name"),
+                        resultSet.getString("license_number"));
+                drivers.add(driver);
```

Comment:
```suggestion
                drivers.add(getDriver(resultSet));
```
Let's extract it into the separate method.